Parity/include/QwQPD.h

```diff
@@ -46,16 +47,17 @@ class QwQPD : public VQwBPM {
     InitializeChannel(subsystemname, name);
     fQwQPDCalibration[0] = 1.0;
     fQwQPDCalibration[1] = 1.0;
-  };    
+  };
   QwQPD(const QwQPD& source)
   : VQwBPM(source),
-    fPhotodiode(source.fPhotodiode),
-    fRelPos(source.fRelPos),
-    fAbsPos(source.fAbsPos),
     fEffectiveCharge(source.fEffectiveCharge)
-  { }
+  {
+    QwCopyArray(source.fPhotodiode, fPhotodiode);
+    QwCopyArray(source.fRelPos, fRelPos);
+    QwCopyArray(source.fAbsPos, fAbsPos);
```

Comment:
You don't need to both change to `std::array` and write an explicit element-wise copy. Since `std::array` is an aggregate type, it does a copy at the element level already on copy construction.

[godbolt](https://godbolt.org/#g:!((g:!((g:!((h:codeEditor,i:(filename:'1',fontScale:14,fontUsePx:'0',j:1,lang:c%2B%2B,selection:(endColumn:2,endLineNumber:8,positionColumn:2,positionLineNumber:8,selectionStartColumn:2,selectionStartLineNumber:8,startColumn:2,startLineNumber:8),source:'%23include+%3Carray%3E%0A%23include+%3Ciostream%3E%0Aint+main(int+argc,+char**+argv)+%7B%0A++++std::array%3Cint,2%3E+a%7B1,2%7D%3B%0A++++std::array%3Cint,2%3E+b(a)%3B%0A++++std::cout+%3C%3C+b%5B0%5D+%3C%3C+%22+%22+%3C%3C+b%5B1%5D+%3C%3C+std::endl%3B%0A++++return+0%3B%0A%7D'),l:'5',n:'1',o:'C%2B%2B+source+%231',t:'0')),k:33.333333333333336,l:'4',n:'0',o:'',s:0,t:'0'),(g:!((h:compiler,i:(compiler:clang1810,filters:(b:'0',binary:'1',binaryObject:'1',commentOnly:'0',debugCalls:'1',demangle:'0',directives:'0',execute:'0',intel:'0',libraryCode:'0',trim:'1',verboseDemangling:'0'),flagsViewOpen:'1',fontScale:14,fontUsePx:'0',j:1,lang:c%2B%2B,libs:!(),options:'',overrides:!(),selection:(endColumn:1,endLineNumber:1,positionColumn:1,positionLineNumber:1,selectionStartColumn:1,selectionStartLineNumber:1,startColumn:1,startLineNumber:1),source:1),l:'5',n:'0',o:'+x86-64+clang+18.1.0+(Editor+%231)',t:'0')),k:33.333333333333336,l:'4',n:'0',o:'',s:0,t:'0'),(g:!((h:output,i:(compilerName:'x86-64+clang+18.1.0',editorid:1,fontScale:14,fontUsePx:'0',j:1,wrap:'1'),l:'5',n:'0',o:'Output+of+x86-64+clang+18.1.0+(Compiler+%231)',t:'0')),k:33.33333333333333,l:'4',n:'0',o:'',s:0,t:'0')),l:'2',n:'0',o:'',t:'0')),version:4)